.changeset/odd-bobcats-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fake-scope/fake-pkg": patch
```

Comment:
cc @tdeekens 

I think it's a bug in the changesets github app

Reply:
https://github.com/commercetools/merchant-center-application-kit/commit/ce14eef2a5c3891441eccc7da0b9ccf41f5dd245